src/components/SearchResultDetail/SearchResultDetail.tsx

```diff
@@ -4,16 +4,42 @@ import * as React from 'react';
 
 import ClipLoader from 'react-spinners/ClipLoader';
 
-import { findSearchResultAction } from '@/app/actions/action';
+import { findSearchResultAction, exportSearchResultPdfAction } from '@/app/actions/action';
 import { SearchResultStatus } from '@/constants/enum';
 import { useModal } from '@/hooks/useModal';
 import toast from '@/lib/toast';
 
 import SearchResultHtmlModal from './SearchResultHtmlModal';
 
-const SearchResultDetail = ({ id }: { id: string }) => {
+interface DownloadBase64Params {
+  base64String: string;
+  fileName: string;
+  mimeType: string;
+}
+
+const mimeTypePDF = 'application/pdf';
```

Comment:
That should be in SCREAMING_SNAKECASE:

```suggestion
const MIME_TYPE_PDF = 'application/pdf';
```

I also noticed an inconsistent casing (that was missed) in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/blob/develop/src/constants/constants.ts#L8 and https://github.com/nimblehq/ic-nextjs-prabesh-dahal/blob/develop/src/constants/constants.ts#L23. This  is just an FYI so no expectations in fixing these in this PR.

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/133/commits/4104aed84bae4716a0216f199ee913b00c57d662 🙈 .Sorry. I will be more careful on constants